cajita/src/Cajita_DynamicPartitioner.hpp

```diff
@@ -575,11 +477,11 @@ class SparseDimPartitioner : public BlockPartitioner<NumSpaceDim>
 
     /*!
       \brief optimize the partition in three dimensions seperately
-      \param is_changed label if the partition is changed after the optimization
       \param iter_seed seed number to choose the starting dimension of the
       optimization
+      \param is_changed label if the partition is changed after the optimization
     */
-    void optimizePartition( bool& is_changed, int iter_seed )
+    void optimizePartitionAlongDim( int iter_seed, bool& is_changed )
```

Comment:
`updatePartition`? This does one update in the optimization loop along all dimensions, right?